### ISSUE TITLE
Doctrine\\DBAL\\Driver\\PDOStatement contains 2 abstract methods and must therefore be declared abstract or implement the remaining methods 

### ISSUE THREAD

**Issue by cpuzzuol** [doctrine/dbal#3284]
#### Summary

I just upgraded from version 2.3 to 2.8 and have the same issue in both cases. This is included in my composer file for a Laravel 5.6 install running PHP 7.1.21.

Every once in a while, I will get a barrage of errors into my apache log:

PHP Fatal error:  Class Doctrine\\DBAL\\Driver\\PDOStatement contains 2 abstract methods and must therefore be declared abstract or implement the remaining methods (Doctrine\\DBAL\\Driver\\Statement::errorCode, Doctrine\\DBAL\\Driver\\Statement::rowCount) in /home/emutoday/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php on line 35

And in fact, the PDOStatement class does not implement these methods errorCode() and rowCount(), which are present in the Statement interface (lines 90 and 129, respectively)

Our server currently seems to be having a problem with segfaults, and these errors occasionally occur after these errors crop up. I'm not sure if this is the exact cause of these segfaults though, as it doesn't happen every time.

#### Expected behaviour

Basically to not see these errors.

**Comment by morozov** [comment]
Closing this as a duplicate of https://github.com/doctrine/dbal/issues/2847 — same error, reproducible with Laravel.

@cpuzzuol, please make sure you have PDO loaded.
